enhancements/etcd/etcd-tuning-profiles.md

```diff
@@ -0,0 +1,178 @@
+authors:
+  - "@dusk125"
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - "@hasbro7, etcd team"
+  - "@tjungblu, etcd team"
+  - "@williamcaban, Openshift product manager"
+  - "@deads2k, implemented a similar feature for API server"
+approvers: # A single approver is preferred, the role of the approver is to raise important questions, help ensure the enhancement receives reviews from all applicable areas/SMEs, and determine when consensus is achieved such that the EP can move forward to implementation.  Having multiple approvers makes it difficult to determine who is responsible for the actual approval.
+  - "@hasbro7, etcd team"
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - None
+creation-date: 2023-05-16
+last-updated: 2023-07-27
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/ETCD-425
+---
+
+# etcd Tuning Profiles
+
+## Summary
+
+This enhancement would replace the hardcoded values for the etcd parameters HEARTBEAT_INTERVAL and LEADER_ELECTION_TIMEOUT with predefined "profiles".
+Each profile would map to predefined, and pretested, values for the internal etcd parameters.
+This would allow for some user tweaking without giving them access to the full range of values.
+This enhancement only covers the mvp for a tech preview release of this new feature; a future enhancement will be necessary.
+
+## Motivation
+
+Customers have asked for the ability to change the etcd heartbeat interval and leader election timeout to aid in the stability of their clusters.
+We want to be able to allow for this while minimizing the risk of them setting values that cause issues.
+This will also remove the hardcoded platform-dependent values; certain platforms could have different default profiles to maintain backwards compatibilty with the currently hardcoded values.
+
+### User Stories
+
+* As an adminstrator, I want to change etcd tuning profiles to help increase the stability of my cluster and understand the performance/latency cost of the profile change.
+* As an Openshift support, I want to walk a customer through changing the active etcd profile in a minimal number of steps.
+* As an etcd engineer, I want to easily add and test new profiles and internal profile parameters.
+
+### Goals
+
+* Add the currently known profiles:
+  * Default (""):
+    - HEARTBEAT_INTERVAL: Platform dependent
+    - LEADER_ELECTION_TIMEOUT: Platform dependent
+  * Standard:
+    - HEARTBEAT_INTERVAL: 100ms
+    - LEADER_ELECTION_TIMEOUT: 1000ms
+  * "Slower" (or other name):
+    - HEARTBEAT_INTERVAL: 500ms
+    - LEADER_ELECTION_TIMEOUT: 2500ms
```

Comment:
This should go in the proposal section since this is outlining the actual implementation.
Also can you outline the platform dependent values (or link to the ones we have today) so we have them here for reference on how they differ from the slower/standard profiles.